packages/legacy/core/App/components/misc/CredentialCard10.tsx

```diff
@@ -38,7 +38,7 @@ const { width } = Dimensions.get('window')
  | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
  | 2 |   |   |   |   |   |   |   |
  | 3 |   |   |   |   |   |   |   |
- | 4 |   |   |   |   |   |   |   |
+ | 4 |   |   |   |   |   |   |   |1
```

Comment:
Extra 1 here

Reply:
might be a missclick while changing this file. Will revert that line.